docs/current/quickstart/snippets/step3/dagger.mjs

```diff
@@ -0,0 +1,21 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  const source = await client.container()
+      .from("node:16")
+      .withMountedDirectory('/src', client.host().directory('.'))
+      .withExec(['git', 'clone', 'https://github.com/vikram-dagger/hello-dagger'])
+      .id()
+
+  const runner = client
+    .container({ id: source })
+    .withWorkdir("./hello-dagger")
+    .withExec(["npm", "install"])
```

Comment:
This can be refactored as follows:

```suggestion
   const source = client.container()
    .from("node:16")
    .withMountedDirectory('/src', client.host().directory('.'))
    .withExec(['git', 'clone', 'https://github.com/vikram-dagger/hello-dagger'])


  const runner = source.withWorkdir("./hello-dagger")
    .withExec(["npm", "install"])
```

Reply:
Modified